### ISSUE TITLE
Suggestion: Ars Énergistique

### ISSUE THREAD

**Issue by 62832** [EnigmaticaModpacks/Enigmatica9#704]
### URL

https://www.curseforge.com/minecraft/mc-mods/ars-energistique

### Why would you like the mod added?

Full disclaimer: I am the developer of the mod being suggested. I developed this as an add-on for integration between Ars Nouveau and Applied Energistics 2 in order to allow for dedicated storage and transfer of Source within ME networks and further augment automation between the two mods. Should you be interested, it might make for a good addition to an Enigmatica pack.

A future build of this mod will also provide Spell P2P Tunnels as an embedded ME alternative to Spell Prisms, alongside other small integration features such as certus quartz support for the Amethyst Golem and (soon) the ability for Scribes Tables to automatically draw items from ME storage via Interfaces.

**Comment by Kanzaji** [comment]
Duplicate of #700, however let's wait on Motoko's reponse on this one because of new features 😅
